src/lib/x509/x509_ext.h

```diff
@@ -750,8 +750,6 @@ class BOTAN_PUBLIC_API(3, 9) ASBlocks final : public Certificate_Extension {
 
             ASIdOrRange() = default;
 
-            ASIdOrRange(asnum_t id) : m_min(id), m_max(id) {}
-
             ASIdOrRange(asnum_t min, asnum_t max) : m_min(min), m_max(max) {
                if(max < min) {
                   throw Decoding_Error("AS range numbers must be sorted");
```

Comment:
I would love to make these constructors private too, but that clashes both with our helper methods in `x509_ext.cpp` as well as with `BER_Decoder`